Sources/Sentry/SentrySubClassFinder.m

```diff
@@ -24,115 +25,73 @@ - (instancetype)initWithDispatchQueue:(SentryDispatchQueueWrapper *)dispatchQueu
     return self;
 }
 
-- (void)actOnSubclassesOf:(Class)parentClass block:(void (^)(Class))block
+- (void)actOnSubclassesOfViewControllerInImage:(NSString *)imageName block:(void (^)(Class))block;
 {
     [self.dispatchQueue dispatchAsyncWithBlock:^{
```

Comment:
`m`: Do we know how fast this is now? Maybe we don't need to move this to a background thread anymore?

Reply:
we need to test this in a project with a couple hundred classes.

Reply:
You can use https://github.com/getsentry/sentry-cocoa/blob/master/scripts/generate-classes.sh for that.

Reply:
Well, for a project with 1007 View controllers, it took ~140ms in debug to find and swizzle all classes.
But only 10ms on the "find" part, and 70ms on the swizzling part, the rest is because we queue the swizzling part to run on the main thread.

With 100 view controller, the total is 70ms, 5ms to find and 3ms to swizzle.

For comparison, the previous approach, with the same 100 view controllers, took ~90ms total, 12ms to find, 11ms to swizzle.

Reply:
We have to swizzle on the main thread because swizzling on a background thread can lead to crashes, see GH-1355.

With your discoveries which approach do you think works best?

Reply:
Let's keep it the way it is.
At the end of the run cycle, it doesn't make much difference.